notes.xsl

```diff
@@ -2,6 +2,9 @@
 <xsl:template name="note">
 	<xsl:choose>
 		<xsl:when test="@id = 'data-doesnt-accept-undefined'">
+			<code>undefined</code> is not recognised as a data value. Calls such as <code><xsl:value-of select="@data-title" />( <xsl:value-of select="@data-parameters" />, undefined )</code> will return the jQuery object that it was called on, allowing for chaining.
```

Comment:
> recognised

We use American English. Could you fix that on the while in the `jquery-data-doesnt-accept-undefined` note.